src/components/GamePanel.vue

```diff
@@ -1,18 +1,25 @@
 <template>
+  <div id="overlay" v-if="allowReset" @click="manualReset"></div>
   <div id="MemoryPanel">
     <div id="gridContainer" class="gloss">
```

Comment:
kebab-case is preferred as id

Reply:
also for classes?

Reply:
As far as I know, there is nothing uniform, we have not established anything uniform, but I would do it that way. But only my opinion, you do not have to take over